docs/IE_DG/supported_plugins/AutoPlugin_Debugging.md

```diff
@@ -0,0 +1,136 @@
+# Debugging Auto-Device Plugin {#openvino_docs_IE_DG_supported_plugins_AUTO_debugging}
+
+## Debug log
+In case of execution problems, just like all other plugins, Auto-Device provides the user with information on exceptions and error values. If the returned data is not enough for debugging purposes, more information may be acquired by means of LOG_LEVEL.
+
+There are six levels of logs, which can be called explicitly or set via the OPENVINO_LOG_LEVEL  environment variable (can be overwritten by compile_model or set_config):  
+0. LOG_NONE 
+1. LOG_ERROR 
+2. LOG_WARNING
+3. LOG_INFO
+4. LOG_DEBUG
+5. LOG_TRACE
+
+@sphinxdirective
+.. tab:: C++ API
+
+   .. code-block:: cpp
+
+      ov::runtime::Core core
+      
+      // read a network in IR, PaddlePaddle, or ONNX format
+      std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
+      
+      // load a network to AUTO and set log level to debug
+      ov::runtime::CompiledModel compiled_model = core.compile_model(model, “AUTO”, {{"LOG_LEVEL ", "LOG_DEBUG"}});
+      
+      // or set log level with set_config and load network
+      core.set_config({"LOG_LEVEL", "LOG_DEBUG"}}, "AUTO");
+      ov::runtime::CompiledModel compiled_model = core.compile_model(model, “AUTO”);
```

Comment:
we should not put the code directly to the documentation, we need to put it into C++ file (which is compiled during the build and thus - the correctness of the code is checked) and use doxygen `@snippet` here

BTW, `ov::runtime` should be replaced with `ov::` (see https://github.com/openvinotoolkit/openvino/pull/9781)

Reply:
the primary question is how to interpert the output with the logging enabled, e.g. the example log, adding @wangleis and @wenjiew